superset/utils/pandas_postprocessing/prophet.py

```diff
@@ -134,7 +135,11 @@ def prophet(  # pylint: disable=too-many-arguments
         raise InvalidPostProcessingError(_("DataFrame include at least one series"))
 
     target_df = DataFrame()
-    for column in [column for column in df.columns if column != index]:
+    for column in [
```

Comment:
@villebro I've love your input on this as I'm somewhat perplexed as to how this was working. After reenabling the tests it seemed to be trying to fit non-numerical columns within the `pd.DataFrame`. I couldn't find any options where we explicitly define which columns to produce a forecast for.